examples/custom-prompt-enabled/placeholder-values

```diff
@@ -0,0 +1,29 @@
+# [REQUIRED] The namespace where RHDH is installed.
+export DEPLOYMENT_NAMESPACE=my-namespace
+
+# [REQUIRED] This is the image registry where your built road-core/service image is located.
+export RCS_IMAGE=quay.io/my-registry/road-core-service:latest
+
+# [REQUIRED] This is the name of your Backstage CR that is located in the $DEPLOYMENT_NAMESPACE.
+export BACKSTAGE_CR_NAME=ai-rh-developer-hub
+
+# [OPTIONAL] If you want to use a custom prompt file set to 'true'.
+# If the value is unset or false it will ignore custom prompts.
+export USE_CUSTOM_PROMPT=true
```

Comment:
better set the default value to be false. since RHDH lightspeed prompt now is sent through API request. no need to specifically enable prompt file override by default

Reply:
same for other examples